.github/workflows/rust-tests-from-artifacts.yml

```diff
@@ -0,0 +1,141 @@
+name: Rust Tests from Artifacts
```

Comment:
We already have a the `try-runtime` workflow to run the _rust tests_ only and allow to pass the `run-id` or looking for the latest completed one. I can make the changes to use 
`just ahm rust-test $NETWORK $ARTIFACT_DIR_ABS` there too.

Reply:
oh shoot, I didn't see it. Yeah, I think that is good enough. I will close this one.

Reply:
Where is it exactly? I can only find `migration-test-common` which is doing the right thing, but for TS tests.

Reply:
Ahh was lost on some pr, I can push here or in a new pr. Sorry about the noise.